types/mailchimp__mailchimp_transactional/index.d.ts

```diff
@@ -0,0 +1,177 @@
+         * If you specify a pool that does not exist, your default pool will be used instead.
+         */
+        ip_pool?: string;
+
+        /**
+         * When this message should be sent as a UTC timestamp in YYYY-MM-DD HH:MM:SS format.
+         * If you specify a time in the past, the message will be sent immediately.
+         */
+        send_at?: string;
+
+        /**
+         * The information on the message to send
+         */
+        message: {
+            html?: string;
+            text?: string;
+            subject?: string;
+            from_email?: string;
+            from_name?: string;
+            to: Recipient[];
+            headers?: Record<string, string>;
+            important?: boolean;
+            track_opens?: boolean;
+            track_clicks?: boolean;
+            auto_text?: boolean;
+            auto_html?: boolean;
+            inline_css?: boolean;
+            url_strip_qs?: boolean;
+            preserve_recipients?: boolean;
+            view_content_link?: boolean;
+            bcc_address?: string;
+            tracking_domain?: string;
+            signing_domain?: string;
+            return_path_domain?: string;
+            merge?: boolean;
+            merge_language?: 'mailchimp' | 'handlebars';
+            global_merge_vars?: MergeVar[];
+            merge_vars?: RecipientMergeVar[];
+            tags?: string[];
+            subaccount?: string;
+            google_analytics_domains?: string[];
+            google_analytics_campaign?: string;
+            metadata?: Metadata & { website: string };
+            recipient_metadata?: RecipientMetadata[];
+            attachments?: Attachment[];
+            images?: Image[];
+        };
+    }
+
+    /**
+     * HTTP Status 200
+     * An array of objects for each recipient containing the key "email" with the email address,
+     * and details of the message status for that recipient
+     */
+    type SendMessageResponse = Array<{
+        /**
+         * The email address of the recipient
+         */
+        email: string;
+
+        /**
+         * The sending status of the recipient Possible values: "sent", "queued", "scheduled", "rejected", or "invalid".
+         */
+        status: SendingStatus;
+
+        /**
+         * The reason for the rejection if the recipient status is "rejected"
+         * Possible values: "hard-bounce", "soft-bounce", "spam", "unsub", "custom", "invalid-sender", "invalid", "test-mode-limit", "unsigned", or "rule".
+         */
+        reject_reason?: SendRejectReason;
+
+        /**
+         * The message's unique id
+         */
+        _id: string;
+    }>;
+
+    class Messages {
+        send(body: SendMessageRequest): Promise<SendMessageResponse | Error>;
+    }
+
+    class ApiClient {
+        messages: Messages;
+        setDefaultOutputFormat(format: OutputFormat): void;
+    }
+}
+
+declare function initMailchimp(apiKey: string): MailChimpTransactional.ApiClient;
+export = initMailchimp;
```

Comment:
```suggestion
export = instance;
```

Reply:
@peterblazejewicz Thanks for taking the time to check this PR!